### PR TITLE
Adding 3 New Playbooks to Defender Threat Intelligence solution

### PR DESCRIPTION
**Pull request by ajkallur**
Adding 3 New Playbooks to Defender Threat Intelligence solution

MDTI-PassiveDNS
MDTI-PassiveDNSReverse
MDTI-Trackers

   Required items, please complete
   
   Change(s):
   - See guidance below

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Comment by ajkallur** [comment]
@rahul0216/@manishkumar1991  Hi , let me know if any more details needed for this PR. Also, had a quick question, will the playbooks get added to the Content Hub solution as well?

File: Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-PassiveDnsReverse/readme.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Our Reverse DNS data includes the following:
+- Value: the value of the Reverse DNS record.
+- First Seen: a timestamp of the date that we first observed this record on the artifact.
+- Last Seen: a timestamp of the date that we first observed this record on the artifact.
+- Type: the type of infrastructure associated with the record. Potential options include Mail Servers (MX), text files (TXT), name servers (NS), CNAMES, and Start of Authority (SOA) records.
+- Tags: any tags applied to this artifact in the Defender TI system..
```
Comment (rahul0216) [inline Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-PassiveDnsReverse/readme.md:10]: Correct "system.." to "system."

File: Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-Trackers/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,642 @@
+{
+    "$schema":  "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion":  "1.0.0.0",
+    "metadata":  {
+        "comments": "Get Trackers data the Microsoft Sentinels Incident based on MDTI TRackers data.",
```
Comment (rahul0216) [inline Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-Trackers/azuredeploy.json:5]: Correct the spelling 'TRackers'.

**Review by rahul0216 — COMMENTED** [review]
1. Please address the inline comments.
2. I think instead of Host Entity, you should use URL entity in all 3 playbooks. MDTI API provide DNS and rDNS resolution of url, so it makes more sense to me.

**Comment by rahul0216** [comment]
@ajkallur, Yes, once PR is approved these new playbooks will be repackaged with previous playbooks in the same solution.

File: Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-PassiveDnsReverse/readme.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# MDTI-Data-Reverse-Dns
+
+## Overview
+This playbook uses the [Microsoft Defender Threat Intelligence](https://learn.microsoft.com/en-us/defender/threat-intelligence/what-is-microsoft-defender-threat-intelligence-defender-ti) Passive Dns Reverse data to automatically enrich incidents generated by Microsoft Sentinel. Leverage this playbook in order to enrich your incidents with [Passive Dns Reverse ](https://learn.microsoft.com/en-us/defender/threat-intelligence/data-sets#reverse-dns) data hosted by the indicators found within the incident. PWhile a forward DNS lookup queries the IP address of a certain hostname, a reverse DNS lookup queries a specific hostname of an IP address. This dataset will show similar results as the DNS dataset. Reviewing DNS records can be helpful to identify shared infrastructure used by actors across the domains they own. For example, actor groups tend to use the same nameservers to segment their infrastructure or the same mail exchange servers to administer their command and control.
```
[the remark this answers is not in the document]
Comment (ajkallur) [reply Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-PassiveDnsReverse/readme.md]: submitted the change

File: Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-PassiveDnsReverse/readme.md (hunk 1 above)
Comment (ajkallur) [reply Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-PassiveDnsReverse/readme.md]: submitted the change

File: Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-Trackers/azuredeploy.json (hunk 2 above)
Comment (ajkallur) [reply Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-Trackers/azuredeploy.json]: submitted the change

**Comment by ajkallur** [comment]
Hi @rahul0216 , the PassiveDNS, ReverseDNS and Trackers MDTI data is always for Host and IP Address Data, that how MDTI as a product works. there is some Domain intelligence as well but for the usecases showcased by these playbooks its always against host and IP data.

**Review by rahul0216 — APPROVED** [review]
Changes look good. Thanks for clarification.

**Comment by ajkallur** [comment]
Thanks Rahul.

---
Regards,
Ajay

[Logo, icon  Description automatically generated]
Ajay Kallur (he/him)
Senior Product Manager
MDEASM & MDTI CxE
Unified Security Intelligence
***@***.******@***.***>

**Pull request merged**